src/content/posts/cfp-workshop-recap.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "CFP Workshop Recap"
+date: 2019-06-28T00:00:00+10:00
+author: "Raphael Haddad"
```

Comment:
Just a heads up, your name will be pulled from the git commit info

Reply:
will fix